config-production.toml

```diff
@@ -221,13 +221,13 @@ url = "https://github.com/mozilla-conduit/review.git"
 
 [[branch_mappings]]
 source_url = "https://github.com/mozilla-conduit/review.git"
-branch_pattern = ".*"
+branch_pattern = ".+"
```

Comment:
Because https://blog.narf.ssji.net/2025/04/30/empty-matches-in-pythons-re-module/